task-1-data-collection/README.md

```diff
@@ -1 +1,16 @@
 # Placeholder
```

Comment:
This readme doesn't say anything about the actual content or motive of this data collection folder, please update accordingly